plugins/search/src/apis.ts

```diff
@@ -35,12 +37,19 @@ class SearchApi {
 
   private async entities() {
     const entities = await this.catalogApi.getEntities();
-    return entities.items.map((result: any) => ({
-      name: result.metadata.name,
-      description: result.metadata.description,
-      owner: result.spec.owner,
-      kind: result.kind,
-      lifecycle: result.spec.lifecycle,
+    return entities.items.map((entity: Entity) => ({
+      name: entity.metadata.name,
+      description: entity.metadata.description || 'No description',
+      owner:
+        typeof entity.spec?.owner === 'string' ? entity.spec?.owner : 'unknown',
+      kind: entity.kind,
+      lifecycle:
+        typeof entity.spec?.lifecycle === 'string'
+          ? entity.spec?.lifecycle
+          : 'unknown',
```

Comment:
same here as above.

One more side note for a later refactor - it seems to me that you will end up rendering mixed result types together in different rows. You actually don't want the string "unknown" here right? For entities that don't have a lifecycle (validly! because it just isn't a thing for them!) you would just want to show an empty cell instead.

TL:DR....: Maybe this should actually just be `lifecycle: entity.spec?.lifecycle` and let it be undefined if that happens to be the case.